.openpublishing.redirection.json

```
@@ -8,6 +8,10 @@
       "source_path": "docs/about/products.md",
       "redirect_url": "/dotnet/standard/components"
     },
+    {
+      "source_path": "docs/architecture/cloud-native/database-per-microservice.md",
+      "redirect_url": "/dotnet/architecture/cloud-native/database-per-microservice"
```

you're redirecting the file to itself. you should also preserve the historical BI data for the renamed article.

```suggestion
      "redirect_url": "/dotnet/architecture/cloud-native/cloud-native-data",
      "redirect_document_id": true
```

---

Ok. Would it make sense to change name from "cloud-native-data.md" to "cloud-data-patterns.md". Naming would be more consistent with previous chapter?

---

Nish wants to back off on the title change for this PR.  Reverted to cloud-native-data. Let's address in separate PR